controller/api/public/grpc_server.go

```diff
@@ -444,7 +445,7 @@ func reqToQueryReq(req *pb.MetricRequest, query string) (telemPb.QueryRequest, e
 	return queryReq, nil
 }
 
-func formatQuery(query string, req *pb.MetricRequest, sumBy string) (string, error) {
+func formatQuery(query string, req *pb.MetricRequest, sumBy string, filterExclusions []string) (string, error) {
```

Comment:
if we're only using this to filter by `controllerNamespace`, recommend we just call it that and add list functionality later if needed:
```go
func formatQuery(query string, req *pb.MetricRequest, sumBy string, controllerNamespace string) 
```